.travis.yml

```diff
@@ -73,6 +73,8 @@ matrix:
       env:
         - TESTMODE=full
         - NUMPYSPEC="--pre --upgrade --timeout=60 -f $PRE_WHEELS numpy"
+        - NPY_USE_BLAS_ILP64=1
+        - DOWNLOAD_OPENBLAS="https://3f23b170c54c2533c070-1c8a9b3114517dc5fe17b7c3f8c63a43.ssl.cf2.rackcdn.com/openblas64_-v0.3.7-manylinux1_x86_64.tar.gz"
```

Comment:
Sanity check: is this OpenBLAS build staying in sync with the regular build automatically?

Reply:
Yes, it was added to the openblas-builder repo.
Actually, the URL isn't needed here, because it's not used below (`openblas_support.py` knows where to get the stuff).